cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -130,23 +122,34 @@ fn update_price_feeds(
     mut deps: DepsMut,
     env: Env,
     info: MessageInfo,
-    data: &Binary,
+    data: &[Binary],
```

Comment:
Pass in an array of price updates to update_price_feeds for consistency with the EVM implementation